src/typed.jl

```diff
@@ -78,7 +79,11 @@ function dispatch_msg(x::JSONRPCEndpoint, dispatcher::MsgDispatcher, msg)
                 end
             end
         else
-            error("Unknown method $method_name.")
+            if is_request # even invalid requests MUST get a response - see Section 5 of the JSON RPC specification.
+                send_error_response(x, msg, -32601, "Unknown method '$method_name'.", nothing)
+            end
+            # TODO: ignoring notifications is legal, so there's no need to crash here on unknown things. However, removing this requires downstream support.
```

Comment:
Support in LS.jl?

Reply:
That's in reference to something about the VSCode extension having built-in support for LS.jl crashing, for telemetry reasons (or so I recall from one of the issues on the LS.jl repo). I'm not familiar with how that works & interacts, but the assumption that the JSONRPC call throws an error is built into LS.jl, yes. Keeping the throwing behavior would need to be moved there, or changed to some other way entirely.